dashboard/of-cloud-dashboard/handler.js

```diff
@@ -110,13 +128,17 @@ module.exports = (event, context) => {
       headers['Content-Type'] = 'text/html';
 
       const isSignedIn = /openfaas_cloud_token=.*\s*/.test(event.headers.cookie);
+      headers["Set-Cookie"] = event.headers.cookie
```

Comment:
What is this doing?

Reply:
I don't follow.

What happens if this is not present?

Reply:
Please remove this line and instead, parse the cookie and look for the login inside the claim, you can use it to build a redirect URL when the path is `/`.